### ISSUE TITLE
Styles: Insufficient color contrast 

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#1137]
![image](https://user-images.githubusercontent.com/1659006/219786931-ee787d31-3b06-4933-b8fb-8a5a71ecfcc9.png)

Color contrast on coral bright and petrol bright backgrounds does not seem to be above 4.5. They should have black text.
